Solutions/Authomize/Analytic Rules/AWS_role_with_admin_privileges.yaml

```diff
@@ -0,0 +1,61 @@
+id: 734c00a0-a95b-44dd-9b69-d926ed44256d
+name: AWS role with admin privileges
+kind: Scheduled
+description: The policy detect the creation of new AWS roles with administrative privileges.the policy configuration allows limiting the policy to specific accounts.
+severity: High
+queryFrequency: 5m
+queryPeriod: 30m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+- InitialAccess
+relevantTechniques:
+- T1078
+query: |-
+  Authomize_v2_CL
+  | extend EventID = id_s
+  | extend Policy = policy_name_s
+  | extend Severity = severity_s
+  | extend Description = description_s
+  | extend Recommendation = recommendation_s
+  | extend URL = url_s
+  | extend Tactics = tactics_s
+  | where Policy contains "AWS role with admin privileges"
+  | project  EventID, Policy, Severity, Description, Recommendation, URL, Category, Tactics
+requiredDataConnectors:
+- connectorId: Authomize
+  dataTypes: [ "Authomize_v2_CL" ]
+incidentConfiguration:
+  createIncident: true
+  groupingConfiguration:
+    enabled: true
+    reopenClosedIncident: false
+    lookbackDuration: 5h
+    matchingMethod: AnyAlert
+    groupByEntities: []
+    groupByAlertDetails: []
+    groupByCustomDetails: []
+eventGroupingSettings:
+  aggregationKind: SingleAlert
+alertDetailsOverride:
+  alertnameFormat: Alert from Authomize - AWS role with admin privileges
+  alertDescriptionFormat: AWS role with admin privileges - The policy detect the creation of new AWS roles with administrative privileges.the policy configuration allows limiting the policy to specific accounts.
```

Comment:
Update to "privileges. The policy configuration"

Reply:
done